react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -172,14 +174,35 @@ class RCTAztecView: Aztec.TextView {
         recognizer.isEnabled = false
     }
 
-    // MARK - View Height: Match to content height
+    // MARK: - View height and width: Match to the content
 
     override func layoutSubviews() {
         super.layoutSubviews()
+        measureWidth()
         fixLabelPositionForRTLLayout()
         updateContentSizeInRN()
     }
 
+    private func textSize(for width: CGFloat) -> CGSize {
+        let containerSize = CGSize(width: width, height: CGFloat.greatestFiniteMagnitude)
+        let container = NSTextContainer(size: containerSize)
+        let storage = NSTextStorage(attributedString: attributedText)
+        let layoutManager = NSLayoutManager()
+        layoutManager.addTextContainer(container)
+        storage.addLayoutManager(layoutManager)
+        let rawSize =  layoutManager.usedRect(for: container).size
+        return rawSize
+    }
+    
+    private func measureWidth() {
```

Comment:
can we rename this method since it is not just measuring, it is also changing things. `adjustWidth` can be a better candidate.

Reply:
Renamed